src/javascript_bindings.pyx

```diff
@@ -29,14 +29,15 @@ cdef class JavascriptBindings:
     cpdef py_void SetFunction(self, py_string name, object func):
         self.SetProperty(name, func)
 
-    cpdef py_void SetObject(self, py_string name, object obj):
+    cpdef py_void SetObject(self, py_string name, object obj,
+                            py_bool allow_properties=False):
```

Comment:
Does this matter? Returning callable() will return a bool, not the method. It could only return method if it is callable, but then the else would be 'None'?  
I feel we might as well just let it return the property value if that's what's asked for. If the end user tries to call it they'll get an exception just like in python.